contrib/windows/build-installer.nsi

```diff
@@ -7,9 +7,13 @@ OutFile "julia-installer.exe"
 SetCompress off
 CRCCheck on
 SetDataBlockOptimize on
-ShowInstDetails show
+ShowInstDetails hide
 RequestExecutionLevel user
-BrandingText "Julia ${Version}"
+BrandingText " "
```

Comment:
Maybe preserve the `${Version}` in `Name`? I think it seems nice to have in the screenshot.

Reply:
It is nice to have the Version in the name. Also, maybe just major and minor, and not patch (which is what we did with the OS X install).

Reply:
So that we are on the same page  `BrandingText` changes  the text highlighted below  

![image](https://user-images.githubusercontent.com/4319522/57347019-d51e4880-711e-11e9-84e9-877820b9c84e.png)

Reply:
I prefer we don't show that text, IMO it doesn't' add anything useful. 

We can instead add a welcome page to the installer with the version number.

Reply:
Sure. Also, I am not sure there is any reason to say "Please wait while Julia is being installed". I mean does the user really have any other option? ;-)

Reply:
Marking as resolved, by adding a new welcome page with the version